Origin: Commit 04fce6f4c166113bc4d5d5c2d97c30352a1cdede
Author: Jon Turney <jon.turney@dronecode.org.uk>
Last-Update: 2021-11-23
Description: [PATCH] meson: Drop spurious 3rd positional argument to
 gnome.compile_resources
 .
 This may become an error with some future meson version:
 .
 src/raven/meson.build:6:0: ERROR: gnome.compile_resources takes exactly 2 arguments, but got 3.
---
 src/raven/meson.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/raven/meson.build b/src/raven/meson.build
index 2830e39a..9394b895 100644
--- a/src/raven/meson.build
+++ b/src/raven/meson.build
@@ -6,7 +6,6 @@ gresource = join_paths(meson.current_source_dir(), 'ui', 'budgie-raven.gresource
 libraven_resources = gnome.compile_resources(
     'budgie-raven-resources',
     gresource,
-    'budgie-daemon.gresource.xml',
     source_dir: join_paths(meson.current_source_dir(), 'ui'),
     c_name: 'budgie_raven',
 )
-- 
2.32.0

